docs/UserGuide.adoc

```diff
@@ -14,31 +14,28 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/AY1920S2-CS2103T-F11-4/main
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+By: `Team F11-4`      Since: `Jun 2016`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 3 (AB3) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB3 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+Cooking Papa is for those who prefer to use a desktop app for managing recipes. More importantly, Cooking Papa is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Cooking Papa can help you plan and cook meals more easily than traditional GUI apps. Interested? Jump to the Section 2, “Quick Start” to get started. Enjoy!
```

Comment:
I think removing the angle brackets breaks the link to section 2.

Reply:
Added the angle brackets back in